src/tester/test-runner.ts

```diff
@@ -68,6 +68,17 @@ export default async function runTests(
     nProcesses: number,
     selection: "all" | "affected" | RegExp,
 ): Promise<void> {
+    const { changedPackages, dependentPackages, allPackages } = await getAffectedPackagesFromDiff(dt, definitelyTypedPath, selection);

```

Comment:
this change is basically an Extract Method, with an associated addition of `allPackages` to the return type.